app/components/Contacts/Network/ChannelItem.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Box, Flex } from 'rebass'
+import { Flex } from 'rebass'
 import { StatusIndicator, Text } from 'components/UI'
 import { BrowseChannelButton } from '.'
 
 const ChannelItem = ({ statusTooltip, status, name, isSelected, onBrowseClick }) => (
   <Flex as="header" alignItems="center" py={2} my={1} mx={3} css={{ cursor: 'pointer' }}>
-    <Box mr={2} className="hint--right" data-hint={statusTooltip}>
+    <Text mr={2} fontSize="xs" className="hint--right" data-hint={statusTooltip}>
       <StatusIndicator variant={status} />
-    </Box>
+    </Text>
     <Text css={{ '&:hover': { opacity: 0.5 } }}>{name}</Text>
```

Comment:
Agree, but this is a little out of scope of this PR. The only reason I touched this file in the first place is because I have adjusted the way the status indicator works and needed to update other places where it was used. I don't want to extend further than I need to into refactoring unrelated code.